doc/htmldoc/templates/index.html

```diff
@@ -11,6 +11,7 @@
  <meta name="description" content="This is the documentation index for the NEST, a simulator for spiking neuronal networks.">
  <meta name="keywords" content="">
  <link rel="stylesheet" href="_static/stylesheets/application.css">
+ <link rel="stylesheet" href="_static/fonts/material-icons.css">
```

Comment:
Was that perhaps intended to be in `nest-simulator/doc/htmldoc/static/fonts` resp. `nest-simulator/doc/htmldoc/static/stylesheets/`? Or did I get that wrong?

Reply:
Ok :+1: